hudi-flink/src/main/java/org/apache/hudi/table/format/FormatUtils.java

```diff
@@ -44,6 +46,28 @@
   private FormatUtils() {
   }
 
+  public static RowKind getRowKind(IndexedRecord record, int index) {
+    if (index == -1) {
+      return RowKind.INSERT;
```

Comment:
Add comment for the method: Returns the RowKind of the given record, never null.
Returns `RowKind.INSERT` when the given field index not found.